<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadDataProgressDialog</class>
 <widget class="QDialog" name="LoadDataProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(79, 79, 79)</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar::chunk {
     background-color:rgb(255, 136, 0);
 }</string>
   </property>
   <property name="value">
    <number>6</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: gray;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProgressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">Color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Progress:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoadName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">Color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Sending...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
